### ISSUE TITLE
CursorResponse

### ISSUE THREAD

**Issue by WagZol** [WagZol/u3eBattleship#8]
{
"shipLeft": {[[2,1], [2,2], [2,3]], true},
"shipRight": {[[2,-1], [2,-2], [2,-3]], false},
"shipUp": {[[2,1], [1,1], [0,1]], true},
"shipDown": {[[2,1], [3,1], [3,2]], true},
"shipSample": [[0,1], [0,2], [0,3]],
"nextPath": "/ship"
}

Azt korábban nem említettem, hogy ezekben a response-okban nem tartom fontosnak a név megadását mert úgyis annak szól a válasz aki az aktuális elérési útvonalat meghívta. Az hogy egy hajó elférése true vagy false, az attól is függ hogy más hajótól mennyire fér el!!

**Comment by bimlas** [comment]
Igen, én is így képzeltem, ez így jónak tűnik. Illetve egy kis javítás / kérdés: akkor a hajók listáját azonosítóval együtt kapom meg ("C2" hajó + a koordinátái)? Mert ha igen, akkor a `shipSample` helyett elég ezt az azonosítót átküldeni és az alapján be tudom lőni, hogy melyik ikont highlight-oljam.

Most nézem, hogy igen, a #6 szerint átadod az azonosítót is, akkor nem is kell ebben az üzenetben a teljes ship sample, elég csak az azonosító.

**Comment by WagZol** [comment]
Jogos, akkor így lesz:)

```
{
"shipLeft": {[[2,1], [2,2], [2,3]], true},
"shipRight": {[[2,-1], [2,-2], [2,-3]], false},
"shipUp": {[[2,1], [1,1], [0,1]], true},
"shipDown": {[[2,1], [3,1], [3,2]], true},
"shipType": "Su",
"nextPath": "/ship"
}
```

Most veszem észre, a kódblokk esetében a tördelés megvan de nem szinezi be a hátteret!

**Comment by bimlas** [comment]
Igen, így jó. A kódblokk alatt erre gondoltam ahigy most küldted, higy monospace fonttal jelenítse meg. :) Színezés asszem így van:

<pre>
```json
Foobar
```
</pre>
